bluebrain/repo-bluebrain/packages/nmodl/package.py

```diff
@@ -55,6 +55,8 @@ class Nmodl(CMakePackage):
     depends_on("py-pyyaml@3.13:", type=("build", "run"))
     depends_on("spdlog")
     depends_on("py-find-libpython", type=("run",))
+    depends_on("py-importlib-resources", type=("run",), when="+python@:3.8")
```

Comment:
Does this really work? Me thinks that should be `when="+python ^python@3.8"`. 3.7 is already EOL!

I'm tempted to say we should just require 3.9+. End of year, that's going to be the minimum supported version anyways.

Reply:
> should be `when="+python ^python@3.8"`

Probably, I read the docs for a whole of 5 seconds beforehand 😅 NMODL doesn't even support 3.7.
Regarding Python 3.9, I have no strong opinions, I don't even have 3.8 on my machine, only 3.9 and later; does anyone actually use NMODL on `master`, but with Spack and 3.8? I see 3.8 is available on BB5, but have no idea what people use...

Reply:
BB5 has a default of 3.11

Ubuntu 20.04 has 3.8 by default, but some of our/NSE's software has already moved on. I would drop 3.8 to keep things simple, and maybe just add a
```
depends_on("python@3.9:")
```
to formalize that.

Reply:
(my Spack on Ubuntu 20.04 builds it's own Python 3.10 to avoid conflicts with the system packages, too)